src/main/java/org/folio/rest/impl/ProxiesForAPI.java

```diff
@@ -49,21 +46,24 @@ public void postProxiesfor(String lang, ProxiesFor entity,
 
     var postgresClient = PgUtil.postgresClient(vertxContext, okapiHeaders);
 
+    final var failureHandler = new FailureHandler(asyncResultHandler, logger,
+      PostProxiesforResponse::respond500WithTextPlain);
+
     userAndProxyUserComboExists(entity, postgresClient)
-      .onFailure(handleFailure(asyncResultHandler))
       .onSuccess(proxyAlreadyExists -> {
         if (Boolean.TRUE.equals(proxyAlreadyExists)) {
           logger.error("Proxy relationship already exists: {}", entity.getId());
           Errors existsError = ValidationHelper.createValidationErrorMessage(
-            "proxyFor", entity.getId(), "Proxy relationship already exists");
+          "proxyFor", entity.getId(), "Proxy relationship already exists");
```

Comment:
Why has this indentation been removed?

Reply:
That is a mistake. Thanks for finding that. I will correct it.